client/my-sites/ads/main.jsx

```diff
@@ -105,6 +114,44 @@ const AdsMain = React.createClass( {
 						{ this.props.wordAdsError }
 					</Notice>
 				}
+				{ this.props.isUnsafe === 'mature' &&
+					<Notice
+						status="is-warning rads__activate-notice"
+						showDismiss={ false }
+						text={ this.translate( 'Your site has been identified as serving mature content. Our advertisers would like to include only family-friendly sites in the programme.' ) }
```

Comment:
Pretty minor, but I think we've been defaulting to `en-US` spellings for our `en` locale. So this would be `program` vs `programme`